packages/quantic/force-app/main/default/lwc/quanticRefineToggle/quanticRefineToggle.css

```diff
@@ -0,0 +1,24 @@
+.refine-modal__action {
+  right: 0;
+}
+
+.refine-modal__footer-button {
+  width: 80%;
+  max-width: 40rem;
+}
+
+.refine-modal__content {
+  width: 100%;
+  max-width: 40rem;
+}
+
+.refine-button_filters-badge {
+  background: var(--lwc-brandAccessible, #0176d3);
+  color: var(--lwc-colorBackgroundAlt, #ffffff);
+  border-radius: 50%;
+  top: -.5rem;
+  right: -.5rem;
```

Comment:
Why do we need negative stuff?

Reply:
We need it for the positioning of the currently active filters badge:
<img width="52" alt="Badge" src="https://user-images.githubusercontent.com/86681870/169144398-d6d160c1-55fd-4402-93a1-61456d4b9076.png">